src/config/MedlemskapConfig.tsx

```diff
@@ -0,0 +1,31 @@
+import { IKategori, ISpørsmål } from '../models/spørsmal';
+
+export const SpørsmålOgSvar: ISpørsmål[] = [
+  {
+    spørsmål_id: 'søkerOppholderSegINorge',
+    tekstid: 'medlemskap.spm.opphold',
+    kategori: IKategori.Medlemskap,
+    svaralternativer: [
+      { value: 'ja', tekstid: 'svar.ja' },
+      { value: 'nei', tekstid: 'svar.nei' },
+    ],
+  },
+  {
+    spørsmål_id: 'søkerBosattINorgeSisteTreÅr',
+    tekstid: 'medlemskap.spm.bosatt',
+    kategori: IKategori.Medlemskap,
+    svaralternativer: [
+      { value: 'ja', tekstid: 'svar.ja' },
+      { value: 'nei', tekstid: 'svar.nei' },
+    ],
+  },
+  {
+    spørsmål_id: 'søkerErFlyktning',
+    tekstid: 'medlemskap.spm.flyktning',
+    kategori: IKategori.Medlemskap,
+    svaralternativer: [
+      { value: 'ja', tekstid: 'svar.ja' },
+      { value: 'nei', tekstid: 'svar.nei' },
```

Comment:
disse to linjene vil sikkert gjenta seg mye, hvor 'ja' og 'svar.ja' er sammen. Bør man gjøre noe for at å standardisere det? f.eks enum https://github.com/navikt/soknad-kontantstotte/blob/master/src/soknad/types.ts

Reply:
Det kan være lurt! @ninakjek 
Hm, har du forslag for å få inn tekstid-en også? Eller kanskje man bare kan gjøre den sjekken i selve koden enn å lagre og hente tekstid ut fra config? 
Så hvis man skal lage svarsalternativer til radiobuttons, mappe svarene, hvis `Svar = JA`, så returner `tekstid: svar.ja`?